Source/CarthageKit/Git.swift

```diff
@@ -136,6 +136,34 @@ extension Submodule: CustomStringConvertible {
 	}
 }
 
+/// Struct to encapsulate global fetch interval cache
+public struct FetchCache {
+	/// Amount of time before a git repository is fetched again. Defaults to 1 minute
```

Comment:
Yeah, it's not great.

This is a short-term fix for a fairly serious issue. It's not pretty, but it'll work until we can refactor to a design that makes more sense.

It's technical debt, pure and simple. But I think it's the good kind of debt: the kind that lets you achieve an important short-term goal.